src/Hudl.Mjolnir/Hudl.Mjolnir.csproj

```diff
@@ -1,12 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <VersionPrefix>3.2.2</VersionPrefix>
-    <FileVersion>3.2.2</FileVersion>
+    <VersionPrefix>3.2.3</VersionPrefix>
+    <FileVersion>3.2.3</FileVersion>
```

Comment:
Will need [Release + Notes](https://github.com/hudl/Mjolnir/releases) once merged. Looks like we also missed the Release for 3.2.2, so we should add that.